configure.ac

```diff
@@ -863,27 +863,24 @@ then
   ])
 fi
 
-if test "$HAVE_GETHOSTBYNAME" != "1"
```

Comment:
Why remove this check? Doesn't this just make the amiga-specific check now instead get run for everyone?

Reply:
That's the disadvantage of this change. What I need is that this check is executed even if gethostbyname is already found as there can be multiple sources for that. This check would additionally enable AmiSSL. However, I can try to find a way to execute it only if we know we are compiling it for Amiga but then always.

Reply:
It is now done if no gethostbyname is found and if AmiSSL is requested.